### ISSUE TITLE
Allow to disable expiration time claim

### ISSUE THREAD

**Issue by beenje** [vimalloc/flask-jwt-extended#105]
I'd like to generate access tokens that never expire (for use in other applications that access the API).

The "exp" claim is optional in PyJWT but not in flask-jwt-extended.

I guess this could be achieved by passing expires_delta=0 or 'never' or something similar.
Would you be interested to integrate this change if I make a pull request?

**Comment by vimalloc** [comment]
I'm not necessarily opposed to adding that functionality, I'm just not sure that it is needed. I wonder if it would be easier just to set the timedelta to something outrageously high that will never reasonably be a problem. Something like:

```python
# Roughly 100 years in the future
app.config['JWT_ACCESS_TOKEN_EXPIRES'] = datetime.timedelta(weeks=5215)
```

Thoughts?

**Comment by vimalloc** [comment]
Released in v3.4.0. Thanks!

**Comment by beenje** [comment]
Thanks for making a new release so quick!
